samples/compose/src/main/java/com/squareup/sample/compose/hellocomposebinding/HelloBinding.kt

```diff
@@ -0,0 +1,29 @@
+package com.squareup.sample.compose.hellocomposebinding
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.squareup.sample.compose.hellocomposebinding.HelloWorkflow.Rendering
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.compose.composedViewFactory
+
+@OptIn(WorkflowUiExperimentalApi::class)
+val HelloBinding = composedViewFactory<Rendering> { rendering, _ ->
+  Text(
+    rendering.message,
+    modifier = Modifier.fillMaxSize()
+      .clickable(onClick = rendering.onClick)
+      .wrapContentSize(Alignment.Center)
+  )
+}
+
+@OptIn(WorkflowUiExperimentalApi::class)
+@Preview(heightDp = 150, showBackground = true)
+@Composable fun DrawHelloRenderingPreview() {
+  // HelloBinding.preview(Rendering("Hello!", onClick = {}))
```

Comment:
Is this commented out so that it doesn't render by default for people viewing it? Or did you mean to uncomment this?

Reply:
It's commented out because I haven't brought the preview stuff in here yet. I need to add todos.